src/style/color-palette-primary.scss

```diff
@@ -0,0 +1,186 @@
+@import './mixins';
+
+// To be able to use colors with alpha, they are written in RGB.
+// Comments with HEX codes in front of the values are used as reference only.
+// Example of use:
+// solid: `color: rgb(var(--contrast-100))`
+// transparent: `color: rgba(var(--contrast-100), 0.5)`
+
+:root {
+    // LimeTechnologies' Brand Colors (Do not have dark/light mode variants)
+    --lime-brand-color-deep-red: 240, 87, 80; // #f05750
```

Comment:
I don't think we should define the same colour names in several files, so if we want to keep all three files, I think the bigger sets should import the smaller ones.

Reply:
It might be better to just define the whole extended set though, and keep the smaller sets to the documentation… I'm not sure… 🤔

Reply:
I had the same idea first, then it felt better to separate them. It all depends on the workflow we will use for importing. It's also easier to open the file and see the entire available spectrum when you need to choose a color (instead of opening this documentations page; which is also why I have put the HEX versions in there for plugins such as Pigment for atom to render the color in the code).

Reply:
Hmm… I was thinking a bit more… I wonder if we shouldn't just ditch the smaller subset files, and just keep the subsets as concepts in the documentation. Because then we can always just add the subset files later, if we feel we actually have any use for them. If we add them to begin with, and then realise we'd prefer to remove them, removing them is a breaking change, which we'd really prefer to avoid when possible.

Reply:
I fully trust your judgement here. Let's remove them and just present them as guides ;) We should then communicate that they shouldn't go wild and pick any color from the extended set, since this will make a clown out of the UI.